### PR TITLE
Use Fable serializer to handle DUs

### PR DESCRIPTION
**Pull request by forki**

### CODE REVIEW

**Comment by forki** [comment]
@alfonsogarciacaro somehow I think this file should go somewhere directly into fable.*

**Comment by alfonsogarciacaro** [comment]
As this is a browser specific API, it cannot go into Fable.Core, maybe Fable.PowerPack?

**Comment by forki** [comment]
yes powerpack sounds good

File: src/Client/Utils.fs [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
-let save key (data: 'T) =
-    Browser.localStorage.setItem(key, JS.JSON.stringify data)
+let inline delete key =
+    Browser.localStorage.removeItem(key)
 
-let delete key =
-    Browser.localStorage.removeItem(key)
\ No newline at end of file
+let inline save<'T> key (data: 'T) =
+    delete key
```
Comment (alfonsogarciacaro) [inline src/Client/Utils.fs]: Why `delete key`? `Browser.localStorage.setItem` will just overwrite if I'm not wrong.

**Comment by alfonsogarciacaro** [comment]
Added in Fable.PowerPack 1.3.2

File: src/Client/Utils.fs (hunk 1 above)
Comment (forki) [reply src/Client/Utils.fs]: Yes that was an attempt to fix a bug that I had. Not needed at all.

**Pull request merged**